lib/sendtext.rb

```diff
@@ -0,0 +1,24 @@
+require 'twilio-ruby'
+
+class Sendtext
+
+  attr_reader :message, :client
+
+  def initialize twillio_klass=Twilio::REST::Client, account_sid, auth_token
```

Comment:
Style/OptionalArguments: Optional arguments should appear at the end of the argument list. (https://github.com/bbatsov/ruby-style-guide#optional-arguments)